lib/Elastica/Suggest/Term.php

```diff
@@ -126,4 +126,16 @@ public function setMaxTermFrequency($max)
     {
         return $this->setParam('max_term_freq', $max);
     }
+
+    /**
+     * Set the number of distance which implementation to use for comparing how similar suggested terms are.
```

Comment:
Could we move this into a separate PR?

Reply:
And this already exists but method is named `setStringDistanceAlgorithm`, probably added in the meantime.